sql/telemetry/fxa_users_daily_v1/query.sql

```diff
@@ -38,11 +36,9 @@ CREATE TEMP FUNCTION udf_contains_tier1_country(
       )
   )
 );
-  --
-  -- This UDF is also only applicable in the context of this query.
-CREATE TEMP FUNCTION udf_contains_registration(
-  x ANY TYPE
-) AS ( --
```

Comment:
But was this this `--` also generated by the same function? The new formatting looks much better.

Reply:
i manually fixed up this formatting